web/src/components/task/task-sidebar/task-sidebar.tsx

```diff
@@ -408,24 +409,37 @@ const TaskSidebar: FC<TaskSidebarProps> = ({ onRedirectAfterFinish, jobSettings,
-                                            items={categories
-                                                ?.filter((el) => el.parent === 'table')
-                                                .map((el) => ({
-                                                    id: el.name,
-                                                    name: el.name,
-                                                    renderLabel: (el: any) => (
-                                                        <span style={{ color: el.metadata?.color }}>
-                                                            {el.name}
-                                                        </span>
-                                                    ),
-                                                    renderName: (s: any) => (
-                                                        <span style={{ color: el.metadata?.color }}>
-                                                            {el.name}
-                                                        </span>
-                                                    )
-                                                }))}
+                                            items={
+                                                categories
+                                                    ?.filter((el) => el.parent === 'table')
+                                                    .map((el) => ({
+                                                        id: el.name,
+                                                        name: el.name,
+                                                        renderLabel: (el: any) => (
+                                                            <span
+                                                                style={{
+                                                                    color: el.metadata?.color
+                                                                }}
+                                                            >
+                                                                {el.name}
+                                                            </span>
+                                                        ),
+                                                        renderName: (s: any) => (
+                                                            <span
+                                                                style={{
+                                                                    color: el.metadata?.color
+                                                                }}
+                                                            >
+                                                                {el.name}
+                                                            </span>
+                                                        )
+                                                    })) as unknown as RadioGroupItem<{
```

Comment:
It's not the right way to do it, since when we set unknown first it allows as to assert any type later. So if API will change it won't show any problems here. However it's better than nothing.